packages/contracts-bedrock/src/L1/L2OutputOracle.sol

```diff
@@ -41,14 +38,6 @@ contract L2OutputOracle is Initializable, ISemver {
     /// @notice An array of L2 output proposals.
     Types.OutputProposal[] internal l2Outputs;
 
-    /// @notice The address of the challenger. Can be updated via reinitialize.
-    /// @custom:network-specific
-    address public challenger;
```

Comment:
We should keep the lowercase interfaces with the uppercase ones marked as deprecated but still public

Reply:
Dang, OK will do.